<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCMapTool</class>
 <widget class="QWidget" name="QGCMapTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="100,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="2" rowspan="2">
    <widget class="QSlider" name="zoomSlider">
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGCMapWidget" name="map" native="true">
         <widget class="QLabel" name="latitudeLabel">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>59</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>LAT:</string>
          </property>
         </widget>
         <widget class="QLabel" name="longitudeLabel">
          <property name="geometry">
           <rect>
            <x>146</x>
            <y>9</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>LON:</string>
          </property>
         </widget>
         <widget class="QLabel" name="fixLabel">
          <property name="geometry">
           <rect>
            <x>283</x>
            <y>9</y>
            <width>51</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>FIX:</string>
          </property>
         </widget>
         <widget class="QLabel" name="satsLabel">
          <property name="geometry">
           <rect>
            <x>419</x>
            <y>9</y>
            <width>75</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>SATS:</string>
          </property>
         </widget>
         <widget class="QLabel" name="hdopLabel">
          <property name="geometry">
           <rect>
            <x>556</x>
            <y>9</y>
            <width>78</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>HDOP:</string>
          </property>
         </widget>
         <widget class="QLabel" name="zoomLabel">
          <property name="geometry">
           <rect>
            <x>693</x>
            <y>9</y>
            <width>80</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>ZOOM:</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGCMapToolBar" name="toolBar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGCMapToolBar</class>
   <extends>QWidget</extends>
   <header>QGCMapToolBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QGCMapWidget</class>
   <extends>QWidget</extends>
   <header>QGCMapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
